bf_cpp_downloads.html

```diff
@@ -280,12 +350,21 @@
                             <tr>
                                 <td><a href="@SOURCE_CODE@" target="_blank"><img
                                         src="images/bf_source_code.png"
-                                        alt="Zipped Source Code" /></a></td>
+                                        alt="Zipped Bio-Formats Source Code" /></a></td>
                                 <td align="center">@SOURCE_CODE_SIZE@</td>
                                 <td>@SOURCE_CODE_BASE@</td>
                                 <td align="center">@SOURCE_CODE_SHA1@ (<a
                                         href="@SOURCE_CODE@.sha1">SHA1</a>)</td>
                             </tr>
+                            <tr>
+                                <td><a href="@SUPERBUILD_SOURCE_CODE@" target="_blank"><img
+                                        src="images/bf_source_code.png"
```

Comment:
Gus made a new icon for this which you haven't added (bf_super-build_source_code.png)